tesler-core/src/main/java/io/tesler/core/util/session/impl/SessionServiceImpl.java

```diff
@@ -219,7 +216,7 @@ public String getSessionIpAddress() {
 
 	@Override
 	public Map<String, Boolean> getResponsibilities() {
-		return sessionCache.getResponsibilities(getSessionUser(), getSessionUserRole());
+		return uiService.getResponsibilities(getSessionUser(),getSessionUserRole());
```

Comment:
Will it affect the performance? It seems the views are cached by separate `UICache` but responsibilities are not.

Reply:
>>Will it affect the performance?
This methods used session scoped SESSION_CAHCE. 
In cluster mode with spring-session-jdbc this cache was stored in DB (<=> there was no cache), so performace will be same.

But we can improve perfomance in cluster mode if we'll use this cache without session scope.
I've done it. Now, methods previously cached with SESSION_CAHCE cached again in USER_CACHE which is not scoped to session. 

Trade off here is average RAM size for USER_CACHE (elements are not deleted from cache when user logs out), but when all users log-in at the same time, then both session scoped and current caches will require same RAM. So max RAM size is the same.
 
 In concrete app using tesler and having lots of users one should consider redefining bean 
 @Primary
 @Bean(name = USER_CACHE)
 public Cache sessionCache() {}
 with some cache provider supporting memory limits and auto cache eviction (hazelcast, redis and so on), so USER_CACHE is kept as separate cache from UI_CACHE in this commit